spec/factories/course_exp_records.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :manual_exp_record, class: 'Course::ExpRecord' do
+    creator
+    updater
+    course_user
+    exp_awarded 100
+    reason "EXP for some event"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.